extensions/agroal/runtime-dev/src/main/java/io/quarkus/agroal/runtime/dev/ui/DatabaseInspector.java

```diff
@@ -0,0 +1,376 @@
+                            StringWriter writer = new StringWriter()) {
+                        DatabaseMetaData metaData = connection.getMetaData();
+                        try (ResultSet tables = metaData.getTables(null, null, "%", new String[] { "TABLE" })) {
+                            while (tables.next()) {
+                                String tableName = tables.getString("TABLE_NAME");
+                                exportTable(connection, writer, tableName);
+                            }
+                        }
+
+                        return writer.toString();
+                    } catch (IOException ex) {
+                        throw new UncheckedIOException(ex);
+                    }
+                }
+            } catch (SQLException ex) {
+                throw new RuntimeException(ex);
+            }
+        }
+        return null;
+    }
+
+    private void exportTable(Connection conn, StringWriter writer, String tableName) throws SQLException, IOException {
+        try (Statement stmt = conn.createStatement();
+                ResultSet rs = stmt.executeQuery("SELECT * FROM " + tableName)) {
+
+            ResultSetMetaData metaData = rs.getMetaData();
+            int columnCount = metaData.getColumnCount();
+
+            while (rs.next()) {
+                StringBuilder insertQuery = new StringBuilder("INSERT INTO " + tableName + " (");
+
+                for (int i = 1; i <= columnCount; i++) {
+                    insertQuery.append(metaData.getColumnName(i));
+                    if (i < columnCount)
+                        insertQuery.append(", ");
+                }
+
+                insertQuery.append(") VALUES (");
+
+                for (int i = 1; i <= columnCount; i++) {
+                    Object value = rs.getObject(i);
+                    if (value == null) {
+                        insertQuery.append("NULL");
+                    } else if (value instanceof String || value instanceof Date || value instanceof Timestamp) {
+                        insertQuery.append("'").append(value.toString().replace("'", "''")).append("'");
+                    } else {
+                        insertQuery.append(value.toString());
+                    }
+                    if (i < columnCount)
+                        insertQuery.append(", ");
+                }
+
+                insertQuery.append(");\n");
+                writer.write(insertQuery.toString());
+            }
+        }
+    }
+
+    private boolean sqlIsValid(String sql) {
+        if (sql == null || sql.isEmpty())
+            return false;
+        if (allowSql) {
+            return true;
+        } else {
+            String lsql = sql.toLowerCase().trim();
+            return lsql.startsWith("select")
+                    && !lsql.contains("update ")
+                    && !lsql.contains("delete ")
+                    && !lsql.contains("insert ")
+                    && !lsql.contains("create ")
```

Comment:
Sorry, this can be stupid question but I didn't try to run this new cool feature yet, what happens if I do `select product_create from products`, this this `contains("create ")` matches?